static/app/views/insights/agents/components/aiSpanList.tsx

```diff
@@ -266,12 +265,6 @@ function getNodeInfo(node: AITraceSpanNode, colors: readonly string[]) {
     color: colors[1],
   };
 
-  if (isTransactionNode(node)) {
```

Comment:
No need to look for `'transaction.op'` specifically in value, we can use the `baseNode.op` getter